alf/utils/common.py

```diff
@@ -993,7 +974,7 @@ def set_random_seed(seed):
         The seed being used if ``seed`` is None.
     """
     if seed is None:
-        seed = int(np.uint32(hash(str(os.getpid()) + '|' + str(time.time()))))
+        seed = hash(str(os.getpid()) + '|' + str(time.time()))
```

Comment:
It is hard to guarantee that other envs (not gym) can accept a negative random seed.